row_011_capture.py

```
@@ -0,0 +1,330 @@
+            logger.error(f"Capture file for capture_uuid={capture_uuid} not found! Cannot process capture.")
+            raise HTTPException(status_code=500, detail=f"Capture file for capture_uuid={capture_uuid} not found! Cannot process capture.")
+
+        # TODO: If the server dies in the middle of an upload or before /process_capture is called,
+        # this will not work well because the in-memory conversation detection state will be gone.
+        # However, users can protentially re-process the conversation manually.
+
+        # Conversation detection service
+        detection_service: ConversationDetectionService = app_state.conversation_detection_service_by_id.get(capture_uuid)
+        if detection_service is None:
+            logger.error(f"Internal error: No conversation detection service exists for capture_uuid={capture_uuid}")
+            raise HTTPException(status_Code=500, detail="Internal error: Lost conversation service")
```

Ruff (0.13.2)

Abstract `raise` to an inner function

(TRY301)